lib/game_team.rb

```diff
@@ -37,6 +37,30 @@ def self.winningest_coach
   def self.worst_coach
   end
 
+  def self.best_offense
+    #grouped by team_id with keys being the team_id and values is an array of games
```

Comment:
I personally love comments, I think it helps the reader understand whats going on. I was scolded by the instructors for having them on a past project, but I think it might be worth it on this project.

Reply:
Yeah I agree as your team mate this is super helpful for me to understand and therefor explain later! Thanks for this!

Reply:
upvote